generator.go

```diff
@@ -139,6 +161,94 @@ func (p *property) readFromStruct(t reflect.Type) {
 	}
 }
 
+func (p *property) addValidatorsFromTags(tag *reflect.StructTag) {
+	switch p.Type {
+	case "string":
+		p.addStringValidators(tag)
+	case "number", "integer":
+		p.addNumberValidators(tag)
+	}
+}
+
+// Some helper functions for not having to create temp variables all over the place
+func int64ptr(i interface{}) *int64 {
+	j := reflect.ValueOf(i).Convert(reflect.TypeOf(int64(0))).Interface().(int64)
```

Comment:
Please create common variables  the beginning of the file for reflect types (int64, float64)
```
var rTypeInt64, rTypeFloat64 = reflect.TypeOf(int64(0)), reflect.TypeOf(float64(0))

```
Then in this method please first check if given interface is convertible to the destination you want, like `if value.Type().ConvertibleTo(rTypeInt64) ...`